src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout.cs

```diff
@@ -80,31 +82,312 @@ public IList<MenuFlyoutItemBase> Items
+			areOpenCloseAnimationsEnabled = AreOpenCloseAnimationsEnabled;
+
+			if (!areOpenCloseAnimationsEnabled)
+			{
+				return;
+			}
+
+			// On Threshold, we use the MenuPopupThemeTransition for MenuFlyouts.
+			double openedLength = 0;
+
+			// UNO TODO
+			//if (m_tpMenuPopupThemeTransition == null)
+			//{
+			//	Transition spMenuPopupChildTransition;
+			//	spMenuPopupChildTransition = PreparePopupThemeTransitionsAndShadows(pPopup, 0.5 /* closedRatioConstant */, 0 /* depth */);
+			//	m_tpMenuPopupThemeTransition = spMenuPopupChildTransition;
+			//}
+
+			openedLength = ((Control)GetPresenter()).ActualHeight;
+
+			// UNO TODO
+			// (m_tpMenuPopupThemeTransition as MenuPopupThemeTransition).OpenedLength = openedLength;
+			var direction = (placementMode == MajorPlacementMode.Top) ? AnimationDirection.Bottom : AnimationDirection.Top;
+
+			// UNO TODO
+			// (m_tpMenuPopupThemeTransition as MenuPopupThemeTransition).Direction = direction);
+		}
+
+
+		///*static*/
+		//Transition PreparePopupThemeTransitionsAndShadows(
+		//	 Popup popup,
+		//	double closedRatioConstant,
+		//	uint depth)
+		//{
+		//	Transition spTransition;
+		//	TransitionCollection spTransitionCollection;
+		//	MenuPopupThemeTransition spMenuPopupChildTransition;
+
+		//	*transition = null;
+
+		//	(ctl.make(&spMenuPopupChildTransition));
+		//	(ctl.make(&spTransitionCollection));
+
+		//	spMenuPopupChildTransition.ClosedRatio = closedRatioConstant;
+
+		//	FrameworkElement overlayElement;
+		//	overlayElement = popup.OverlayElement;
+		//	spMenuPopupChildTransition.SetOverlayElement(overlayElement);
+
+		//	spTransition = spMenuPopupChildTransition;
+		//	spTransitionCollection.Append(spTransition));
+
+		//	// UNO TODO Windowed Popups are not supported
+		//	//
+		//	// For windowed popups, the transition needs to target the grandchild of the popup.
+		//	// Otherwise, the transition LTE is going to live in the main window and get clipped.
+		//	//if ((CPopup*)(popup.GetHandle()).IsWindowed())
+		//	//{
+		//	//	UIElement popupChild;
+		//	//	popupChild = popup.Child;
+
+		//	//	if (popupChild)
+		//	//	{
+		//	//		int childrenCount;
+		//	//		(VisualTreeHelper.GetChildrenCountStatic((UIElement*)(popupChild), &childrenCount));
+
+		//	//		if (childrenCount == 1)
+		//	//		{
+		//	//			xaml.IDependencyObject popupGrandChildAsDO;
+		//	//			(VisualTreeHelper.GetChildStatic((UIElement*)(popupChild), 0, &popupGrandChildAsDO));
+
+		//	//			if (popupGrandChildAsDO)
+		//	//			{
+		//	//				UIElement popupGrandChildAsUE;
+		//	//				(popupGrandChildAsDO.As(&popupGrandChildAsUE));
+		//	//				(popupGrandChildAsUE.Transitions = spTransitionCollection);
+		//	//				(popupGrandChildAsUE.InvalidateMeasure());
+		//	//			}
+		//	//		}
+		//	//	}
+		//	//}
+		//	//else
+		//	{
+		//		popup.ChildTransitions = spTransitionCollection;
+		//	}
+		//}
+
+		void AutoAdjustPlacement(MajorPlacementMode pPlacement)
+		{
+			// UNO TODO
+			// Rect windowRect = default;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)